src/base/bittorrent/torrentimpl.cpp

```diff
@@ -2388,7 +2388,8 @@ void TorrentImpl::handleFileCompletedAlert(const lt::file_completed_alert *p)
 
 #if defined(Q_OS_MACOS) || defined(Q_OS_WIN)
     // only apply Mark-of-the-Web to new download files
-    if (Preferences::instance()->isMarkOfTheWebEnabled() && isDownloading())
+    if (Preferences::instance()->isMarkOfTheWebEnabled()
+        && ((m_state == TorrentState::ForcedDownloading) || (m_state == TorrentState::Downloading)))
```

Comment:
IIRC, it is still incorrect.
I believe it should the following:
```suggestion
        && (m_nativeStatus.state == lt::torrent_status::downloading))
```

Reply:
OK, changed.